packages/next/src/build/webpack/plugins/next-types-plugin.ts

```diff
@@ -134,38 +134,36 @@ function createRouteDefinitions() {
   let routeTypes = ''
 
   edgeRouteTypes.forEach((route) => {
-    routeTypes += ` | ${route}\n`
+    routeTypes += `  | ${route}\n  `
```

Comment:
```suggestion
    routeTypes += `  | ${route}\n`
```

Reply:
It seems that my PR has been merged by the bot, so I will just mark this as resolved...